packages/next-app-dir-sample/CHANGELOG.md

```diff
@@ -0,0 +1,189 @@
+# Change Log
```

Comment:
Is this changelog copied from `next-sample`? Do we need to keep these history records?

Reply:
removed